applications/fov-quicklook/templates/db.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: quicklook-db
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: quicklook-db
+  template:
+    metadata:
+      labels:
+        app: quicklook-db
+    spec:
+      initContainers:
+        - name: init-permissions
+          image: 'busybox'
+          command: ['sh', '-c', 'chown -R 999:999 /var/lib/postgresql/data']
```

Comment:
I'm a little concerned that this will cause delays on startup.  If nothing else is touching the Postgres data files, can this be limited to doing a `chown` of the mount path only instead of having `-R`?

Reply:
fixed in d387624e31b0ca80102a078524a935a6ee9d899a